Tests/ReactiveSwiftTests/SwiftConcurrencyTests.swift

```diff
@@ -109,7 +109,7 @@ class SwiftConcurrencyTests: XCTestCase {
 	}
 }
 // Extension to allow Throw assertion for async expressions
-@available(macOS 12, iOS 15, watchOS 8, tvOS 15, *)
+@available(macOS 12, iOS 15, watchOS 8, tvOS 15, macCatalyst 15, *)
```

Comment:
Doesn't this need to be added in the Signal and SP extensions also?

Reply:
It's everywhere now, but it somehow still doesn't want to build